actions/templates.go

```diff
@@ -137,37 +132,35 @@ func PostTemplate(c *gin.Context) {
 		return
 	}
 
-	store, err := templates.NewSesTemplateStore(keys.AccessKey, keys.SecretKey, keys.Region)
+	// todo ask about text part and html part should we change now post params ?
+	templateInput := &entities.Template{
+		UserID:      u.ID,
+		Name:        body.Name,
+		HTMLPart:    body.Content,
+		TextPart:    body.Content,
```

Comment:
@FilipNikolovski should we change PostTemplate params? to include html_part + text_part? since it's going to be different now

Reply:
@Dzalevski yes we should include both in the params.